astropy/utils/console.py

```diff
@@ -86,6 +40,33 @@ class PyreadlineConsole(object): pass
 _DEFAULT_ENCODING = 'utf-8'
 
 
+def _get_out_stream():
+    try:
+        from IPython import get_ipython
+    except ImportError:
+        pass
+    try:
```

Comment:
Note that the `ImportError` is passed through above, so this case presumably captures some ipython version where `get_ipython` is always defined, but not in the `IPython` module. I have no idea whether that is a version that still exists...  Anyway, this part is just kept the same as before...